pkg/rpcclient/unwrap/unwrap.go

```diff
@@ -180,6 +180,53 @@ func SessionIterator(r *result.Invoke, err error) (uuid.UUID, result.Iterator, e
 	return r.Session, iter, nil
 }
 
+// ArrayAndSessionIterator expects correct execution (HALT state) with one or two stack
+// items returned. If there is 1 item, it must be an array. If there is a second item,
+// it must be an iterator. This is exactly the result of smartcontract.CreateCallAndPrefetchIteratorScript.
+// NeoGo server SessionsEnabled parameter must be set to true.
+func ArrayAndSessionIterator(r *result.Invoke, err error) ([]stackitem.Item, uuid.UUID, result.Iterator, error) {
+	if err := checkResOK(r, err); err != nil {
+		return nil, uuid.UUID{}, result.Iterator{}, err
+	}
+	if r.State != vmstate.Halt.String() {
+		return nil, uuid.UUID{}, result.Iterator{}, fmt.Errorf("invocation failed: %s", r.FaultException)
+	}
```

Comment:
No need to check it, `checkResOK` does the thing.

Reply:
Fixed